File: gnu/gcc/libgomp/testsuite/libgomp.c/appendix-a/a.26.1.c
/* { dg-do run } */

#include <stdio.h>
int
main ()
{
  int i, j;
  i = 1;
  j = 2;
#pragma omp parallel private(i) firstprivate(j)
  {
    i = 3;
    j = j + 2;
  }
  printf ("%d %d\n", i, j);	/* i and j are undefined */
  return 0;
}

File: gnu/gcc/libgomp/testsuite/libgomp.fortran/appendix-a/a.26.1.f90
! { dg-do run }
       PROGRAM A26
         INTEGER I, J
         I=1
         J=2
!$OMP PARALLEL PRIVATE(I) FIRSTPRIVATE(J)
           I=3
           J=J+2
!$OMP END PARALLEL
          PRINT *, I, J ! I and J are undefined
      END PROGRAM A26
